airflow/www/static/js/main.js

```diff
@@ -287,3 +287,18 @@ $(document).ready(() => {
   // Global Tooltip selector
   $(".js-tooltip").tooltip();
 });
+
+$(".reparse_dag").click((event) => {
+  console.log(event);
+  event.preventDefault();
+  $.ajax({
+    url: event.currentTarget.attributes.href.value,
+    type: "PUT",
+    done(response) {
+      console.log(response);
```

Comment:
Hey @bbovenzi, how do we usually inform users about the success/failures in the UI? I saw a flash methods but it's for Flask and since we are only making a ajax call to API Flask don't come into picture.

Reply:
Yes, we haven't called the REST API from FAB pages before. Only in react and FAB endpoints had their own actions in `views.py`.

We probably should wrap the API call inside of a function in `views.py` so we can still use the FAB alerts. Hopefully we can remove that soon when we migrate these pages to react.

Reply:
Thanks, created a wrapper view around REST API as per the suggestions. PTAL.